src/test/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtilTest.java

```diff
@@ -311,19 +353,26 @@ public void test_shouldCreatePageableObjectWithDefaultPageSize() {
     }
 
     @Test
-    public void test_getShowDeletedValueTrue() {
-        String showDeleted = "True";
-        String response = RefDataUtil.getShowDeletedValue(showDeleted);
-
-        assertThat(response.equals("true")).isTrue();
+    public void test_getShowDeletedValue() {
+        assertThat(RefDataUtil.getShowDeletedValue("True")).isEqualTo("true");
+        assertThat(RefDataUtil.getShowDeletedValue("true")).isEqualTo("true");
+        assertThat(RefDataUtil.getShowDeletedValue("TRUE")).isEqualTo("true");
+        assertThat(RefDataUtil.getShowDeletedValue("False")).isEqualTo("false");
+        assertThat(RefDataUtil.getShowDeletedValue("false")).isEqualTo("false");
+        assertThat(RefDataUtil.getShowDeletedValue("FALSE")).isEqualTo("false");
+        assertThat(RefDataUtil.getShowDeletedValue("invalid")).isEqualTo("false");
+        assertThat(RefDataUtil.getShowDeletedValue("")).isEqualTo("false");
+        assertThat(RefDataUtil.getShowDeletedValue(" ")).isEqualTo("false");
     }
 
     @Test
-    public void test_getShowDeletedValueFalse() {
-        String showDeleted = "false";
-        String response = RefDataUtil.getShowDeletedValue(showDeleted);
-
-        assertThat(response.equals("false")).isTrue();
+    public void test_getReturnRolesValue() {
+        assertThat(RefDataUtil.getReturnRolesValue(Boolean.valueOf("True"))).isEqualTo(true);
+        assertThat(RefDataUtil.getReturnRolesValue(Boolean.valueOf("true"))).isEqualTo(true);
+        assertThat(RefDataUtil.getReturnRolesValue(Boolean.valueOf("TRUE"))).isEqualTo(true);
+        assertThat(RefDataUtil.getReturnRolesValue(Boolean.valueOf("False"))).isEqualTo(false);
+        assertThat(RefDataUtil.getReturnRolesValue(Boolean.valueOf("false"))).isEqualTo(false);
+        assertThat(RefDataUtil.getReturnRolesValue(Boolean.valueOf("FALSE"))).isEqualTo(false);
     }
```

Comment:
why is this  test required? hv not we converted the query parameter to boolean ?

Reply:
yes , its correct , need to change parameter to boolean.

Reply:
Created tech debt for this

Reply:
https://tools.hmcts.net/jira/browse/RDCC-1680